src/shared/remoted_op.c

```diff
@@ -108,6 +107,8 @@ void parse_uname_string (char *uname,
                     match_size = match[1].rm_eo - match[1].rm_so;
                     os_malloc(match_size +1, osd->os_major);
                     snprintf(osd->os_major, match_size + 1, "%.*s", match_size, osd->os_version + match[1].rm_so);
+                } else {
+                    os_strdup("", osd->os_major);
```

Comment:
As this is now a valid case, I would add a unit test case to cover it.

Reply:
Added
![image](https://user-images.githubusercontent.com/13010397/155589422-9d003008-30b1-4f38-a119-695a0ba054db.png)

Reply:
Added
![image](https://user-images.githubusercontent.com/13010397/155589422-9d003008-30b1-4f38-a119-695a0ba054db.png)

Reply:
Added
![image](https://user-images.githubusercontent.com/13010397/155589422-9d003008-30b1-4f38-a119-695a0ba054db.png)